src/operator/tensor/init_op.h

```diff
@@ -272,10 +272,22 @@ inline bool InitShape(const nnvm::NodeAttrs& attrs,
   CHECK_EQ(in_attrs->size(), 0U);
   CHECK_EQ(out_attrs->size(), 1U);
   mxnet::TShape param_shape = param.shape;
+  if (shape_is_known(param_shape) && !features::is_enabled(features::INT64_TENSOR_SIZE)) {
```

Comment:
Why not directly check the compiler flag here?

Reply:
I personally prefer that way - looks cleaner :)

Reply:
in certain cases shape is not know at the beginning, then param_shape.Size() throws a different error, which is not a good user experience. That's why this check is required.

Reply:
@apeforest @marcoabreu do you think this makes sense now ?